app/permissions/PermissionActionCheck.scala

```diff
@@ -13,12 +13,16 @@ trait PermissionActionFilter extends ActionFilter[UserRequest] {
   val restrictedAction: String
 
   override def filter[A](request: UserRequest[A]) =
-    testAccess(request.user.email).map {
-      case PermissionGranted => None
-      case PermissionDenied =>
-        Logger.info(s"user not authorized to $restrictedAction")
-        println("FAILED")
-        Some(Results.Unauthorized)}
+    if(request.user.email == "hmac-authed-service") {
+      Future.successful(None)
+    } else {
+      testAccess(request.user.email).map {
+        case PermissionGranted => None
+        case PermissionDenied =>
+          Logger.info(s"user not authorized to $restrictedAction")
+          println("FAILED")
```

Comment:
Is this println still needed?

Reply:
Seems like no, if there's a log, I'll take it out